Tahi-project/tahi
# Re-building seeds for 'rake db:data:load' to work again

JIRA Issue: There is none. This was fixed while working on APERTA-5690 as it was a blocker, but it's being made its own PR in case this needs to get merged in first for others who may be blocked.

This updates the DB seeds to work now that the `manuscripts` table is gone from `schema.rb`. 
#### How to test
- Pull down this branch
- Migrate your database (in case you have any pending)
- Run `rake db:data:load`

It should succeed without fail.
#### Here's what I did

_In case I missed a step...._
- Create new branch off master
- Reverted commit f9218b74648ad42d13a6908806b07ff2e40c1ea4
- Ran `rake db:drop && rake db:create && rake db:schema:load && rake db:data:load`
- Opened up `psql` locally and ran `DROP TABLE manuscripts;`. Quit `psql`.
- Ran `rake db:data:dump`
- Committed. Rebased to remove the revert commit added above. Pushed.

Reviewer tasks:
- [x] If I created a migration, I updated the base data.yml seeds file
